recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "v0.6.0" %}
+{% set version = "v0.7.0a4" %}
 
 package:
   name: ifcopenshell
   version: {{ version }}
 
 source:
-  url: https://github.com/IfcOpenShell/IfcOpenShell/archive/{{ version }}.tar.gz
-  sha256: da22910d83472886eb53693ced79f5b229d7898c76a73bc5ac7003a7c913af82
+  git_url: https://github.com/Krande/IfcOpenShell
+  git_tag: pr-daily-builds
+  git_depth: 1
```

Comment:
@looooo I think this part can be reverted back to the original Ifcopenshell repo now that my PR is complete. I am not sure where you find the relevant sha256 tag, but here's the tagged release: https://github.com/IfcOpenShell/IfcOpenShell/releases/tag/v0.7.0a4